### DISCUSSION TITLE
Trying to load a lot of comments can take forever. There are other online tools that handle it better

### DISCUSSION THREAD

**Discussion by Letssharearow** [sonigy/YCS#47] [General]
Esspecially on popular videos with lots of comment the tool is very nice. But if there are say 1 Million or 100k comments it can take a while to load them all. There are tools that can handle this better 

for example https://hadzy.com/
How do they do it and is it posible to do the same for YCS?

**Comment by oldspammer** [comment]
See item https://github.com/sonigy/YCS/discussions/33 that I submitted a while ago.

This slowness on large comment numbers for some videos was caused by displaying the progress messages for every single comment grabbed by his software, but now it does not bother updating as frequently, this has sped up comment loading a lot, but it is still a bit slower than it should be. I do not know if he could do a kind of two-pass approach that streamlines or optimizes the slowest process of network traffic and grabbing of the raw comments so that once in memory he can perform indexing or digesting processing of all words found within the comments on the second pass through the data.
